neo4j/src/test/java/org/hibernate/ogm/datastore/neo4j/test/mapping/UnidirectionalManyToOneTest.java

```diff
@@ -50,35 +53,28 @@ public void prepareDb() throws Exception {
 
 	@Test
 	public void testMapping() throws Exception {
-		assertNumberOfNodes( 3 );
-		assertRelationships( 2 );
-
-		String jugNode = "(jug:JUG:ENTITY {jug_id: {jug}.jug_id, name: {jug}.name })";
-		String emmanuelNode = "(e:Member:ENTITY {member_id: {e}.member_id, name: {e}.name})";
-		String jeromeNode = "(j:Member:ENTITY {member_id: {j}.member_id, name: {j}.name})";
-
-		Map<String, Object> jugProperties = new HashMap<String, Object>();
-		jugProperties.put( "jug_id", jug.getId() );
-		jugProperties.put( "name", jug.getName() );
-
-		Map<String, Object> emmanuelProperties = new HashMap<String, Object>();
-		emmanuelProperties.put( "member_id", emmanuel.getId() );
-		emmanuelProperties.put( "name", emmanuel.getName() );
-
-		Map<String, Object> jeromeProperties = new HashMap<String, Object>();
-		jeromeProperties.put( "member_id", jerome.getId() );
-		jeromeProperties.put( "name", jerome.getName() );
-
-		Map<String, Object> params = new HashMap<String, Object>();
-		params.put( "jug", jugProperties );
-		params.put( "e", emmanuelProperties );
-		params.put( "j", jeromeProperties );
-
-		assertExpectedMapping( "jug", jugNode, params );
-		assertExpectedMapping( "e", emmanuelNode, params );
-		assertExpectedMapping( "j", jeromeNode, params );
-		assertExpectedMapping( "r", jugNode + " - [r:memberOf] - " + emmanuelNode, params );
-		assertExpectedMapping( "r", jugNode + " - [r:memberOf] - " + jeromeNode, params );
+		NodeForGraphAssertions jugNode = node( "jug", JUG.class.getSimpleName(), ENTITY.name() )
```

Comment:
I wanted to know which properties you expected to find in the node and give more information about the properties on the element. In particular I don't want unexpected properties in the node or relationship that I matched. Now that I think about it I don't check the labels.

With the cypher query, you only know if the pattern you are looking for has a match or not.